### ISSUE TITLE
I can catch pokemons. but can't loot any pokestops

### ISSUE THREAD

**Issue by icozmo** [finndev/PokeBuddy#957]
Hello fellows.. I used pokebuddy for a couple hours.. went back finding that I ran out of pokeballs. I can catch pokemons but I can't loot any pokestops. what does that mean?

**Comment by futi666** [comment]
same here, can catch pokemons but cant farm pokestops, i also tried to farm pokestops on nox but there is the same problem

**Comment by icozmo** [comment]
Yes, It keeps saying "try again later"

**Comment by futi666** [comment]
maybe server problems from niantic

**Comment by icozmo** [comment]
I don't think so. I've got a friend who plays legally and everything works fine with him

**Comment by futi666** [comment]
hmh... this is the first time i have this issue, hopefully it is just a bug or something

**Comment by icozmo** [comment]
Let's hope so.

**Comment by jonson09** [comment]
i have the same problem, but friends of mine are using the bot now normaly. when mine is running their bot is buggy. don´t know, i think there is a lot of request at the pokebuddyserver. at the beginning all my bots could run at the same time, i had them in several VMs. but i noticed when 1 was running und i startet some more, pokebuddy was not working any more, my suggestion is, that there is now a limit, one pokebuddyinstance/PKMgo login per ip adress. i tryed diffrent vpn to change ip and geotag, but no success, think they flag the PKMgo login once, then you lost. only do´nt know if niantic or finndev is the reason, both have reason to...  
Yesterday i also tried Nox, also don´t work, but reason was old PKMgo version.

Seas futi666, nit so fein wenn nit gscheid geht, schwer zu sagen warum... hab schon viel ausprobiert, i denk  entweder es is zwecks niantic, eher aber noch dass da botserver selber, aufgrund zu vieler user uns vielnutzern a auszeit einräumt und wenn mehrere anstartet draht er di a glei ab, oder niantic, naja wie a immer i glab jz hab i genug deitsch gschrieben
habedere

**Comment by futi666** [comment]
@jonson09  sers, i hab pokemon go mal auffn nox app player gstartet, und da is das gleiche problem, also i kann pokemon normal fangen aber bei den pokestops steht, versuche es später nocheinmal

**Comment by jonson09** [comment]
aha, des hab i noch nit probiert/khabt i log mal per nox player ein.

**Comment by futi666** [comment]
ist bei mir auch das erste mal...

**Comment by uglysoft** [comment]
Same problem. Around 12 hours ago my PC running Nox with the GPS set to Minneapolis suddenly stopped being able to spin pokestops for exp. The error was "Try again later".

Pokemon catching and gym attacks still work, though, so it's not the typical softban.

If this lasts for another 12 hours I'm going to run the game on my iPhone at the real location, wait out the softban (i.e. another 12 hours) and see if I can loot pokestops at my real location.

This ban appears to be something new. Let's find out how it works!

**Comment by jonson09** [comment]
i go on holiday for some days, also can not loot stops.

**Comment by valak666** [comment]
I'm experiemcing that problem right now. Are there any solutions? Please help
